projects/addon-mobile/components/tab-bar/tab-bar.template.html

```diff
@@ -0,0 +1,7 @@
+<ng-content></ng-content>
+<div class="t-skeletons">
+    <div
+        *tuiRepeatTimes="let index of 4"
```

Comment:
`4` - can we override this value?

if we know that we always have 5 items

<img width="436" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/29fdda66-369b-4912-90f6-ecd743f653e7">